src/Symfony/Component/DependencyInjection/Tests/Fixtures/Prototype/Foo.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Symfony\Component\DependencyInjection\Tests\Fixtures\Prototype;
 
+use Symfony\Component\DependencyInjection\Attribute\When;
+
+#[When(env: 'prod')]
```

Comment:
We should have a test using the attribute multiple times, to show that it properly does a OR

Reply:
Agreed, updated.